src/Core/src/Platform/iOS/MauiUIContextMenuInteraction.cs

```diff
@@ -14,15 +15,19 @@ class MauiUIContextMenuInteraction : UIContextMenuInteraction
 		WeakReference<IElementHandler> _handler;
 
 		// Store a reference to the platform delegate so that it is not garbage collected
-		IUIContextMenuInteractionDelegate? _uiContextMenuInteractionDelegate;
+		[UnconditionalSuppressMessage("Memory", "MEM0002", Justification = "Delegate is weak on Objective-C side")]
+		readonly IUIContextMenuInteractionDelegate? _uiContextMenuInteractionDelegate;
```

Comment:
I'm not quite sure what this does, but assuming it's sorta similar to what was there before (that is, holding onto the delegate for as long as required), then that sounds good to me.

Reply:
It's the same as before, we just made it `readonly` and `[UnconditionalSuppressMessage("Memory", "MEM0002"]` is the thing that will silence this analyzer:

https://github.com/jonathanpeppers/memory-analyzers

There might be better designs/ideas for this analyzer.